### PR TITLE
Update MS teams docs

### PR DESCRIPTION
**Pull request by merit-maita**
## Status
- [ ] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
updated the docs to support xsoar 8

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams_description.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 Use the Microsoft Teams integration to send messages and notifications to your team members and create meetings. 
-Note: Currently, this integration does not work with Cortex XSOAR 8.2 and up without using an engine.
+Note: the integration is supported in Cortex XSOAR 8 without using an engine.
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams_description.md:2]:
```suggestion
Note: This integration is supported in Cortex XSOAR 8 when not using an engine.
```

File: Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md [hunk 2]
```diff
@@ -2,7 +2,7 @@ Use the Microsoft Teams integration to send messages and notifications to your t
 This integration was integrated and tested with version 1.0 of Microsoft Teams.
 
 **Note:** 
-- Currently, this integration is supported in Cortex XSOAR 8 and up without using an engine.
+- This integration is supported in Cortex XSOAR 8 and up without using an engine.
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md:5]:
```suggestion
- This integration is supported in Cortex XSOAR 8 when not using an engine.
```

File: Packs/MicrosoftTeams/ReleaseNotes/1_4_49.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Microsoft Teams
+
+- Documentation and metadata improvements.
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftTeams/ReleaseNotes/1_4_49.md:6]:
```suggestion
Documentation and metadata improvements.
```

**Comment by ShirleyDenkberg** [comment]
@michal-dagan @thefrieddan1 Doc review completed.

File: Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md (hunk 2 above)
Comment (merit-maita) [reply Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md:5]: @ShirleyDenkberg in this note, i'm saying that its supported in xsoar 8 with no need to an engine (but they can use an engine), so i guess "when not" is not the right expression

Comment (ShirleyDenkberg) [reply Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md:5]: ok

**Review by thefrieddan1 — APPROVED** [review]
LGTM

**Pull request merged**